R/pkg/inst/tests/testthat/test_mllib_regression.R

```diff
@@ -77,6 +77,18 @@ test_that("spark.glm and predict", {
   out <- capture.output(print(summary(model)))
   expect_true(any(grepl("Dispersion parameter for gamma family", out)))
 
+  # tweedie family
+  require(statmod)
```

Comment:
we can't require this as of now - we would need to update Jenkins otherwise it would fail like it is right now:
```
spark.glm and predict (@test_mllib_regression.R#81) - there is no package called 'statmod'

```